docs/BR.md

```diff
@@ -1,20 +1,14 @@
-**CA/Browser Forum**
-
-**Baseline Requirements**
-**for the**
-**Issuance and Management of**
-**Publicly-Trusted Certificates**
-
-**CA/Browser Forum**
-
-**Version 1.7.3**
-
-**October 19, 2020**
-
-**www.cabforum.org**
-
-Copyright 2020 CA/Browser Forum
-This work is licensed under the Creative Commons Attribution 4.0 International license.
+---
```

Comment:
Thanks to https://github.com/isaacs/github/issues/1655, leaving some comments here at the top of the file:

* There are only four places where there are two blank lines in a row: lines 32, 41, 1856, and 2278. Seem out of place, might as well remove them.
* There are many places (distributed nearly at random, not even always between sentences) where two consecutive spaces are used instead of just one.
* There are many instances of trailing whitespace at the ends of lines.
* Section 7.2.2 subsection 1 (CRL Extensions: Reason Code) is only indented two spaces, when it needs to be indented 3 spaces in order to properly nest under the ordered list item. (This is actually my least favorite thing about markdown: the amount you have to indent changes depending on the width of the list item marker.)
* The example in A.1.2 is not surrounded in a code block.
* Line 1671 is not indented to nest under its bullet point.
* A slew of "Heading should be surrounded by whitespace" markdownlints.

I've fixed these in my own branch, and am about to open a PR against this branch so you can see if you want to bring them in to this ballot.

Reply:
> Thanks to [isaacs/github#1655](https://github.com/isaacs/github/issues/1655), leaving some comments here at the top of the file:
> 
> * There are only four places where there are two blank lines in a row: lines 32, 41, 1856, and 2278. Seem out of place, might as well remove them.
> * There are many places (distributed nearly at random, not even always between sentences) where two consecutive spaces are used instead of just one.

Yeah, I was a little nervous about touching this. I noticed different members have different styles when proposing ballots, and since we don't really have a way to enforce consistency, I didn't think to pick this up.

> * There are many instances of trailing whitespace at the ends of lines.

These are actually semantically meaningful, in that they force the alignment of sections as a [manual line break](https://pandoc.org/MANUAL.html#paragraphs)

> * Section 7.2.2 subsection 1 (CRL Extensions: Reason Code) is only indented two spaces, when it needs to be indented 3 spaces in order to properly nest under the ordered list item. (This is actually my least favorite thing about markdown: the amount you have to indent changes depending on the width of the list item marker.)
> * The example in A.1.2 is not surrounded in a code block.
> * Line 1671 is not indented to nest under its bullet point.

Thanks for spotting these!

> * A slew of "Heading should be surrounded by whitespace" markdownlints.

I didn't know there was a linter. It looks like it's [not aware of](https://github.com/DavidAnson/markdownlint/issues/76) Pandoc's extension of CommonMark (which we use to get the docx/LaTeX files looking right), but I'd be curious to try and see about getting it on C-I if it's not too noisy.

> I've fixed these in my own branch, and am about to open a PR against this branch so you can see if you want to bring them in to this ballot.

SG!

Reply:
PR here: https://github.com/sleevi/cabforum-docs/pull/34

> These are actually semantically meaningful, in that they force the alignment of sections as a manual line break

WHAT. Well I guess I'll go look at the PR and see if I broke any of these. That's completely wild, though.